tests/unit/test-util/test.desktop-notifier.js

```diff
@@ -22,13 +21,12 @@ describe('util/desktop-notifier', () => {
         notifier: fakeNotifier,
       })
         .then(() => {
-          assert.ok(fakeNotifier.notify.called);
-          assert.equal(
-            fakeNotifier.notify.firstCall.args[0].title,
-            'web-ext run: title',
+          sinon.assert.calledWith(
```

Comment:
I forgot to mention that there is a shortcut you can use called [`sinon.assert.calledWithMatch()`](http://sinonjs.org/releases/v2.3.5/assertions/)